c-api/tests/src/test_doctype_api.c

```diff
@@ -109,6 +109,48 @@ static void test_get_user_data(void *user_data) {
     );
 }
 
+//-------------------------------------------------------------------------
+EXPECT_OUTPUT(
+    remove_doctype_output_sink,
+    "<!DOCTYPE><html></html>",
```

Comment:
shouldn't this have been removed?
```suggestion
    "<html></html>",
```

Reply:
😅 A copy pasta error!